azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/utils/AppServiceUtils.java

```diff
@@ -390,7 +391,7 @@ public static FunctionEntity fromFunctionAppEnvelope(@Nonnull FunctionEnvelope f
                 .entryPoint(entryPoint)
                 .scriptFile(scriptFile)
                 .bindingList(bindingEntities)
-                .functionAppId(functionEnvelope.functionAppId())
+                .functionAppId(functionApp.id())
```

Comment:
why?

Reply:
RestAPI will not return functionAppId in list request, which will make `functionEnvelope.functionAppId()` returns null